### PR TITLE
Feature/documentation tweaks

### PR DESCRIPTION
**Pull request by albertshau**

### CODE REVIEW

File: docs/docs/source/guide/admin/server-config.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ Configuring the server
 ----------------------
 
 Loom server uses Zookeeper for task coordination and a database for persistent data. The server will work out of the box
-without any configuration options with an in memory Zookeeper and an embedded Derby DB; however, we 
+without any configuration options with an in process Zookeeper and an embedded Derby DB; however, we
```
Comment (dereklwood) [inline docs/docs/source/guide/admin/server-config.rst]: pretty sure it should be in-process (with a hyphen)

File: docs/docs/source/guide/user/index.rst [hunk 2]
```diff
@@ -118,10 +119,68 @@ accessed using the corresponding IP addresses, usernames and passwords (through
...
+    :align: center
+    :width: 800px
+    :alt: Reconfigure cluster
+    :figclass: align-center
+
+Starting, Stopping, and Restarting Services
+-------------------------------------------
+Services can be started, stopped, and restarted from the cluster description page screen as well. To start a service,
+click on the green triangle next to the service name. To stop a service, click on the red square next to the 
+service name. It restart a service, click on the white circular arrows next to the service. When a start, stop, or
```
Comment (dereklwood) [inline docs/docs/source/guide/user/index.rst]: "To restart a service"...

**Comment by dereklwood** [comment]
couple typos, otherwise lgtm

File: docs/docs/source/guide/admin/server-config.rst (hunk 1 above)
Comment (albertshau) [inline docs/docs/source/guide/admin/server-config.rst]: done

File: docs/docs/source/guide/user/index.rst (hunk 2 above)
Comment (albertshau) [inline docs/docs/source/guide/user/index.rst]: good catch... don't know what happened there

**Pull request merged**